scpd/lms/static/css/vendor/su-identity.css

```diff
@@ -0,0 +1,302 @@
+}
+
+#su-footer img {
+  margin: 0 auto;
+  display: block;
+}
+
+/* Clear the floats */
+#su-footer .clear {
+  clear: both;
+}
+
+/*
+========================================================
+Media queries
+*/
+
+@media screen and (min-width: 480px) {
+}
+
+@media screen and (min-width: 768px) {
+  /* Prevent double background gradient */
+  #su-footer .su-container {
+    background:none;
+    box-shadow:none;
+  }
+
+  #bottom-text {
+    margin-top:2px;
+  }
+
+  /* Remove left padding */
+  #su-header .su-container img {
+    padding:4px 0 0;
+  }
+
+  /* Bootstrap style */
+  #su-header .su-container,
+  #su-footer .su-container {
+    width:724px;
+  }
+
+  #su-footer .su-container .su-row {
+    margin:30px 0 0 -20px;
+  }
+  /* Footer single column */
+  #su-footer .span2 {
+    float:left;
+    width:104px;
+    padding-left:8px; /* Prevent flush left */
+  }
+  /* Bootstrap style */
+  #su-footer .span10 {
+    width:600px;
+    margin-left:20px;
+    margin:0 12px 0 20px;
+    float:left;
+  }
+
+  #su-footer img {
+    margin:0 auto;
+  }
+
+  #su-footer ul li.alt {
+  	clear:right;
+  }
+  /* Footer two column */
+  #su-footer ul {
+    width:100%;
+    float:left;
+  }
+
+  #su-footer ul li {
+  	width:auto;
+  	margin:0 13px 3px 0;
+  	padding:0;
+  	float:none;
+  	clear:left;
+    display:inline-block;
+  }
+
+  #bottom-logo {
+    margin:-5px 0 10px;
+  }
+
+  #su-footer .clear {
+    clear:none;
+  }
+
+  p.copyright {
+    text-align:left;
+    padding-left:17%;
+    padding-top:25px;
+  }
+}
+
+@media screen and (min-width: 980px) {
+  /* Bootstrap style */
+  #su-header .su-container,
+  #su-footer .su-container {
+    width:940px;
+  }
+
+  #su-footer .span2 {
+    width:140px;
+  }
+
+  #su-footer .span10 {
+    width:780px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  /* Bootstrap style */
+  
+  #su-header .su-container,
+  #su-footer .su-container {
+    width:1180px;
+  }
+
+  #su-footer .span2 {
+    width:170px;
+  }
+
+  #su-footer .span10 {
+    width:970px;
+  }
+
+  p.copyright {
+    padding-left:16%;
+  }
+}
+
+/*
+Sticky Footer Solution
+by Steve Hatcher 
+http://stever.ca
+http://www.cssstickyfooter.com
+*/
+
+/*html, body {*/
+	/*height: 100%;*/
+/*}*/
+/*#su-wrap {*/
+	/*min-height: 100%;*/
+/*}*/
+/*#su-content {*/
+	/*overflow:visible;*/
+	/*padding-bottom: 120px; [> must be same height as the footer <]*/
+/*}*/
+/*#su-footer {*/
+	/*position: relative;*/
+	/*margin-top: -120px; [> negative value of footer height <]*/
+	/*height: 120px;*/
+	/*clear:both;*/
+/*} */
+
+/*[> IE6/7 uses media, <]*/
+/*@media, {*/
+  /*#su-footer {*/
+    /*height: 219px;*/
+  /*}*/
+/*}*/
+
+/*[> IE8 uses \0/ <]*/
+/*@media all {*/
+  /*#su-footer {*/
+    /*height: 196px\0/;*/
+  /*}*/
+/*}*/
+
+/*[>Opera Fix<]*/
+	/*body:before { [> thanks to Maleika (Kohoutec)<]*/
+	/*content:"";*/
+	/*height:100%;*/
+	/*float:left;*/
+	/*width:0;*/
+	/*margin-top:-32767px;[> thank you Erik J - negate effect of float<]*/
+/*}*/
```

Comment:
I can't comment on the images below, so I'll do it here.

Things like the images should be added to the todo/follow-up checklist:
- "what image should we use for the logo?"
- "what image should we use for the panoramic background?"
- etc.